buyer-signing-service/src/middlewares/validateSignBody.js

```diff
@@ -0,0 +1,31 @@
+import { isPresent } from '../utils/wibson-lib';
+
+/**
+ * Middleware to check sign body has all needed parameters.
+ */
+const validateSignBody = async (req, res, next) => {
+  const { nonce, gasPrice, params } = req.body;
+  let errors = [];
+
+  if (!isPresent(nonce)) {
+    errors = [...errors, 'Field \'nonce\' is required'];
```

Comment:
Is there a reason for immutability ?

Reply:
Not really, I got used to write it like this.